### ISSUE TITLE
Problems with the 2.9.0 release

### ISSUE THREAD

**Issue by XplittR** [cplotts/snoopwpf#76]
Hello, and thank you for releasing a new version of Snoop, it's such a great utility.

However, the newest 2.9.0 installer does not seem to work properly.

The `setup.exe` simply fails with the following dialog from Windows Installer: 
```This installation package could not be opened.  Verify that the package exists and that you can access it or contact the application vendor to verify that this is a valid Windows Installer package.```
Snoop 2.8.0 was installed, but the same error message was displayed after un-installing 2.8.0.

The `snoop.msi` installer works, but I also want to let you know that I get a notification from Windows Defender SmartScreen preventing the app to run, due to it having an unknown publisher, and I thus had to click "more options" before I was able to run it.

My relevant specs:
Windows 10 Pro
Intel i7 x64

**Comment by cplotts** [comment]
Fascinating. I just added a comment to the release ... but Setup.exe is a bootstrapper that tries to install the appropriate redistributables and then launches Snoop.msi. Many developers won't even need Setup.exe and will only need to install Snoop.msi.

That being said ... it's odd that it fails like that. It's almost like you don't have access to the redists.

**Comment by fdncred** [comment]
Downloaded files like this are blocked by default in the latest Windows. So, I Unblocked them and everything installed fine.

**Comment by cplotts** [comment]
@fdncred How did you unblock them? Can you comment here so that others can benefit? Thanks!

**Comment by fdncred** [comment]
Right click on the exe and then click properties. From there click unblock checkbox and hit apply.
![unblock_2018-08-01_11-00-17](https://user-images.githubusercontent.com/343840/43533351-3d15db00-957a-11e8-9d8d-e19fe3287130.png)

**Comment by drewnoakes** [comment]
I used the MSI to install 2.9.0 on Windows 10 and got a warning that the executable was not trusted/verified or something like that. Such warnings can be sidestepped by clicking something like "more options" and "run anyway" (from memory). I researched this a while ago for one of my own apps and IIRC MS tracks executable signatures. If enough people run the app without problems, it gets whitelisted.

**Comment by cplotts** [comment]
I am closing this issue because I believe that things are working as-designed. It is just that the latest version of Windows blocks files that are downloaded ... and the user needs to unblock these files to allow things to proceed.

I wonder if there is a better way to do things here or if we simply have to wait till the application gets white listed.

Also, if anyone knows how to modify things so that it no longer shows up as an coming from an unknown publisher ... let me know. Installers are not my cup of tea ... in fact, someone else helped me with both Setup.exe and the .wxs file that generates the .msi.

**Comment by aaron-ni** [comment]
Also note that setup.exe is showing up with 8 "virus" detection results on VirustTotal: https://www.virustotal.com/en/file/3667fddeaeeba5a3cdf268710d4d193ef09f6a6ab0b75019a2b38977a20f0444/analysis/

Sometimes these are spurious, but I usually don't see more than 2 or 3.

**Comment by cplotts** [comment]
Yeah, I've looked into this in the past. I don't think it has a virus ... I've done a binary comparison from the original that we committed a long time ago and it is the same.

Regardless, we are going to be getting rid of Setup.exe as part of a host of other changes related to dropping support for WPF 3.5 and upgrading the build process for Snoop entirely to Visual Studio 2017. At some later point, likely the next release, we're also going to create a new installer using a current version of WiX. Simply put ... it's going away soon. Even more, most people don't even need it.

**Comment by mahimabhowmik** [comment]
Snoop not detecting anything when application hosted in browser. Attached the screendhot.
![snooping](https://user-images.githubusercontent.com/42257213/43933986-29d7ca90-9c90-11e8-8a0d-15049eac27ef.PNG)

**Comment by batzen** [comment]
@mahimabhowmik could you create a separate issue for your problem?

**Comment by mahimabhowmik** [comment]
Yes, but could you please let me know a workaround? I am really stuck.. :(

**Comment by mahimabhowmik** [comment]
Created #78 for the problem I faced.

**Comment by cplotts** [comment]
I am curious if Snoop 2.8.0 works ... does it?

**Comment by aaron-ni** [comment]
I did check Setup.exe for Snoop 2.8.0 and it has the same 8 detections on Virus Total (I use sigcheck.exe from Microsoft Sysinternals to quickly check that). Since neither the msi or setup.exe are signed, setup.exe could likely be used to setup other nefarious payloads unless you have some kind of hash check built into it. Getting rid of it sounds good though since the msi is sufficient.

**Comment by cplotts** [comment]
Good point. I'm deleting it now. People can install the redists themselves.
